jobs/kola-aws.Jenkinsfile

```diff
@@ -44,23 +44,32 @@ if (s3_stream_dir == "") {
     s3_stream_dir = "fcos-builds/prod/streams/${params.STREAM}"
 }
 
-cosaPod(image: params.COREOS_ASSEMBLER_IMAGE,
-        memory: "256Mi", kvm: false,
-        secrets: ["aws-fcos-builds-bot-config", "aws-fcos-kola-bot-config"]) {
+try { timeout(time: 120, unit: 'MINUTES') {
+    cosaPod(image: params.COREOS_ASSEMBLER_IMAGE,
+            memory: "256Mi", kvm: false,
+            secrets: ["aws-fcos-builds-bot-config", "aws-fcos-kola-bot-config"]) {
 
-    stage('Fetch Metadata') {
-        shwrap("""
-        export AWS_CONFIG_FILE=\${AWS_FCOS_BUILDS_BOT_CONFIG}/config
-        cosa init --branch ${params.STREAM} https://github.com/coreos/fedora-coreos-config
-        cosa buildprep --ostree --build=${params.VERSION} --arch=${params.ARCH} s3://${s3_stream_dir}/builds
-        """)
-    }
+        stage('Fetch Metadata') {
+            shwrap("""
+            export AWS_CONFIG_FILE=\${AWS_FCOS_BUILDS_BOT_CONFIG}/config
+            cosa init --branch ${params.STREAM} https://github.com/coreos/fedora-coreos-config
+            cosa buildprep --ostree --build=${params.VERSION} --arch=${params.ARCH} s3://${s3_stream_dir}/builds
+            """)
+        }
 
-    fcosKola(cosaDir: env.WORKSPACE, parallel: 5,
-             build: params.VERSION, arch: params.ARCH,
-             extraArgs: params.KOLA_TESTS,
-             skipBasicScenarios: true,
-             platformArgs: """-p=aws \
-                --aws-credentials-file=\${AWS_FCOS_KOLA_BOT_CONFIG}/config \
-                --aws-region=us-east-1""")
+        fcosKola(cosaDir: env.WORKSPACE, parallel: 5,
+                 build: params.VERSION, arch: params.ARCH,
+                 extraArgs: params.KOLA_TESTS,
+                 skipBasicScenarios: true,
+                 platformArgs: """-p=aws \
+                    --aws-credentials-file=\${AWS_FCOS_KOLA_BOT_CONFIG}/config \
+                    --aws-region=us-east-1""")
+    }
```

Comment:
I think a gotcha here is that you have to explicitly set `currentBuild.result = 'SUCCESS'`, because otherwise, the condition in the `finally` block will fail.

Reply:
Same for other files.

Reply:
fixed